src/include/common.h

```diff
@@ -543,7 +522,7 @@ unordered_set<server_thread_t, thread_hash> get_responsible_threads_metadata(
 
   for (auto it = mts.begin(); it != mts.end(); it++) {
     string ip = it->get_ip();
-    auto tids = responsible_local(key, DEFAULT_LOCAL_REPLICATION, local_memory_hash_ring);
+    auto tids = responsible_local(key, 1, local_memory_hash_ring);
```

Comment:
Is this saying that we only replicate metadata once? Shouldn't this also be configurable?